chapters/chap03-sec03.tex

```diff
@@ -114,45 +117,46 @@ \subsection{Instantiating the packing functions}\label{instantiation:statement:p
             \t \t out \gets out +_{\FFx{\rBN}} 2^{\len{x}-1-i} \\
             \pcreturn out;
         }
-        \caption{Packing algorithm in big endian.}\label{zeth-protocol:fig:packing-alg}
+        \caption{Algorithm to pack bits into a field element.}\label{zeth-protocol:fig:packing-alg}
     \end{minipage}%
     \begin{minipage}[t]{.6\textwidth}
         \centering
         \procedure{\packResBits{\nfs{}, \vin, \vout, \hsig, \htags{}}}{%
             out \gets []; r \gets \epsilon; \\
             r \gets \slice{\vin}{\floor{\zvalueLen / \bnFieldBitCap} \cdot \bnFieldBitCap}{}; \\
```

Comment:
Above you have:
```
    \widetilde{\hsig} \concat \widetilde{\nfs{}} \concat \widetilde{\htags{}} \concat \vin \concat \vout
```

Which does not seem consistent with this algo. Seems like you need to invert vin and vout here.

Reply:
Ah yes, `vout` is pushed first.  Fixed. Thanks.